### PR TITLE
make protobuf-import more robust against different worker amounts

### PR DESCRIPTION
**Pull request by guswynn**
### Motivation

  * This PR fixes a previously unreported bug.

This test only worked for a small number of workers. @philip-stoev is there some way to trigger a redpanda run of this?

### Checklist

- [x] This PR has adequate test coverage / QA involvement has been duly considered.
- [x] This PR adds a release note for any [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/user/content/release-notes.md#what-changes-require-a-release-note).

### CODE REVIEW

File: test/testdrive/protobuf-multiple-message.td [hunk 1]
```diff
@@ -13,8 +13,6 @@
 $ set the-schema
 syntax = "proto3";
 
-import "google/protobuf/timestamp.proto";
```
Comment (elindsey) [inline test/testdrive/protobuf-multiple-message.td:16]: Strange that csr was okay with this and rpk's schema registry isn't, any idea what's going on there?

**Review by elindsey — COMMENTED** [review]
lgtm!

https://buildkite.com/materialize/nightlies/builds/540

File: test/testdrive/protobuf-multiple-message.td (hunk 1 above)
Comment (guswynn) [reply test/testdrive/protobuf-multiple-message.td:16]: no idea, its entirely possible that redpanda is running a new version, or with different default flags, references and imports working is not something csr shipped originally with

File: test/testdrive/protobuf-import.td [hunk 2]
```diff
@@ -165,26 +165,18 @@ $ kafka-create-topic topic=import-csr-wrong-proto partitions=1
 $ set-regex match="expected schema id: \d*, found \d*" replacement="expected schema id: X, found X"
 
 # Setup the correct first message, priming all the workers with the correct schema id
-$ kafka-ingest repeat=9 topic=import-csr-wrong-id format=protobuf descriptor-file=import.pb message=.Importer confluent-wire-format=true
+$ kafka-ingest repeat=100 topic=import-csr-wrong-id format=protobuf descriptor-file=import.pb message=.Importer confluent-wire-format=true
```
Comment (quodlibetor) [inline test/testdrive/protobuf-import.td:168]: why did this need to change to 100?

**Review by quodlibetor — APPROVED** [review]

File: test/testdrive/protobuf-import.td (hunk 2 above)
Comment (guswynn) [reply test/testdrive/protobuf-import.td:168]: because we test with a large number of workers in nightly (32), and this makes sure they all fill up

Comment (guswynn) [reply test/testdrive/protobuf-import.td:168]: Would it be worth adding an issue to add like {worker_count} and {worker_count_doubled} to the testdrive env?

Comment (quodlibetor) [reply test/testdrive/protobuf-import.td:168]: yeah plausibly that would be worth having.

Comment (quodlibetor) [reply test/testdrive/protobuf-import.td:168]: it would also make it easier to determine which tests care about how many workers there are.

**Pull request merged**